templates/stanford_event/stanford-event.html.twig

```diff
@@ -2,42 +2,7 @@
 <div class="content-type__media-container right-align-media">
   <div class="content-type__media">
     <figure>{{ featured_image }}</figure>
-    <div class="content-type-date">
-      <div class="content-type-date__actions social-actions">
-        <div class="tweet-popup sharrre" data-title="<i class='fa fa-twitter' aria-hidden='true'></i>" data-text="{{ node_title|render|striptags|trim }}" data-url="{{ url("<current>") }}">
-          <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="tweet-popup">
-            <svg class="i-svg" role="img">
-              <title>{{ 'twitter'|t }}</title>
-              <use xlink:href="#i-twitter" xmlns:xlink="http://www.w3.org/1999/xlink"></use>
-            </svg>
-          </a>
-        </div>
-        <div class="facebook-popup sharrre" data-title="<i class='fa fa-facebook' aria-hidden='true'></i>" data-text="{{ node_title|render|striptags|trim }}" data-url="{{ url("<current>") }}">
-          <a href="https://www.facebook.com/sharer/sharer.php?u={{ url('<current>') }}" class="facebook-popup">
-            <svg class="i-svg" role="img">
-              <title>{{ 'facebook'|t }}</title>
-              <use xlink:href="#i-fb" xmlns:xlink="http://www.w3.org/1999/xlink"></use>
-            </svg>
-          </a>
-        </div>
-        <div class="linkedin-popup sharrre" data-title="<i class='fa fa-linkedin' aria-hidden='true'></i>" data-text="{{ node_title|render|striptags|trim }}" data-url="{{ url("<current>") }}">
-          <a href="https://www.linkedin.com/shareArticle?mini=true&url={{ url('<current>') }}&title={{ node_title|render|striptags|trim|url_encode }}&source=LinkedIn" class="linkedin-popup">
-            <svg class="i-svg" role="img">
-              <title>{{ 'linkedin'|t }}</title>
-              <use xlink:href="#i-linkedin" xmlns:xlink="http://www.w3.org/1999/xlink"></use>
-            </svg>
-          </a>
-        </div>
-        <div class="email-popup">
-          <a href="mailto:?subject={{ 'I wanted you to see this news article'|t }}&amp;body={{ 'Read more about:'|t }} at {{ node_title|render|striptags|trim }} {{ url("<current>") }}" title="{{ 'Share by Email'|t }}">
-            <svg class="i-svg" role="img">
-              <title>{{ 'mail'|t }}</title>
-              <use xlink:href="#i-mail" xmlns:xlink="http://www.w3.org/1999/xlink"></use>
-            </svg>
-          </a>
-        </div>
-      </div>
-    </div>
+
```

Comment:
Not adding: `<div class="getsocial gs-inline-group"></div>` to this?

Reply:
Oops, I will!
However, attaching the library isn't yet working, so need to fix that too.

Reply:
Oh, also we added it further down. It looks weird in the spot we had it at smaller screen sizes.